java/src/jmri/jmrit/decoderdefn/IdentifyDecoder.java

```diff
@@ -33,6 +33,7 @@
  * <li>DIY: (mfgID == 13) CV47 is the highest byte, CV48 is high byte, CV49 is
  * low byte, CV50 is the lowest byte; (CV47 == 1) is reserved for the Czech
  * Republic</li>
+ * <li>Doehler & Haass: (mfgID == 97) CV261 is ID from 2020 firmwares</li>
```

Comment:
These comments are Javadoc and when the Javadoc is parsed, it's converted to HTML. So a `&` must be written as `&amp;`.